addons/website_slides/models/slide_slide.py

```diff
@@ -208,6 +209,14 @@ class Slide(models.Model):
         ('exclusion_html_content_and_url', "CHECK(html_content IS NULL OR url IS NULL)", "A slide is either filled with a document url or HTML content. Not both.")
     ]
 
+    @api.model
+    def _default_image(self):
+        path = 'website_slides/static/src/img/channel-training-default.jpg'
+        with file_open(path, 'rb') as f:
```

Comment:
```suggestion
        with file_open('website_slides/static/src/img/channel-training-default.jpg', 'rb') as f:
```
We plan to remove the ci/security on static path